photon-lib/src/main/native/cpp/photon/PhotonCamera.cpp

```diff
@@ -200,9 +200,33 @@ void PhotonCamera::VerifyVersion() {
           cameraNameOutString);
     }
   } else if (!VersionMatches(versionString)) {
-    FRC_ReportError(frc::warn::Warning,
+    std::string bfw = "";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+    bfw += ">>>    ____  _________    ____     ________  ___________    __  \n";
+    bfw += ">>>   / __ \\/ ____/   |  / __ \\   /_  __/ / / /  _/ ___/   / /\n";
+    bfw += ">>>  / /_/ / __/ / /| | / / / /    / / / /_/ // / \\__ \\   / / \n";
+    bfw += ">>> / _, _/ /___/ ___ |/ /_/ /    / / / __  // / ___/ /  /_/    \n";
+    bfw += ">>>/_/ |_/_____/_/  |_/_____/    /_/ /_/ /_/___//____/  (_)     \n";
+    bfw += ">>>                                                             \n";
+    bfw += ">>> You are running an incompatible version                     \n";
+    bfw += ">>> of PhotonVision on your coprocessor!                        \n";
+    bfw += ">>>                                                             \n";
+    bfw += ">>> This is neither tested nor supported.                       \n";
+    bfw += ">>> You MUST update either PhotonVision, PhotonLib, or both.    \n";
+    bfw += ">>>                                                             \n";
+    bfw += ">>> Your code will now crash. We hope your day gets better.     \n";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+    bfw += ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n";
+
+    FRC_ReportError(frc::warn::Warning, "\n\n\n\n");
+    FRC_ReportError(frc::warn::Warning, bfw);
+    FRC_ReportError(frc::warn::Warning, "\n\n");
```

Comment:
These will show up as three separate warnings in the log, which is probably not what you want.